src/main/resources/templates/datewarehouse/etl-jobs-config.json

```diff
@@ -271,19 +683,52 @@
 		"sourcePath" : "log.sagebase.org/fileDownloadRecords/",
 		"tableDescriptor": {
 			"name": "filedownloadrecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"user_id": "bigint",
-				"project_id": "bigint",
-				"file_handle_id": "bigint",
-				"association_object_id": "bigint",
-				"association_object_type": "string",
-				"stack": "string",
-				"instance": "string"
-			},
-			"partitionKeys": {
-				"record_date": "date"
+			"description" : "The table contain records of all downloads of Synapse.The events are recorded only after the pre-signed url for requested download entity is generated.",
+			"columns":[{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when file download event is pushed to the queue for recording, after generating the Pre-signed url."
+			},
+			{
+				"name": "user_id",
+				"type" : "bigint",
+				"comment" : "The id of the user who downloaded the file."
+			},
+			{
+				"name": "project_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the project where the downloaded entity resides. Applicable only for FileEntity and TableEntity."
 			},
+			{
+				"name": "file_handle_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the file handle."
+			},
+			{
+				"name": "association_object_id",
+				"type" : "bigint",
+				"comment" : "The unique identifier of the Synapse object ( with out prefix syn) that wraps the file."
+			},
+			{
+				"name": "association_object_type",
+				"type" : "string",
+				"comment" : "The type of the Synapse object that wraps the file."
```

Comment:
List of object types?

Reply:
There is no link for types and we can not add the list in description because description can be only of 255 characters. So I added example . The type of the Synapse object that wraps the file, e.g, FileEntity, TableEntity, WikiAttachment, WikiMarkdown, UserProfileAttachment, MessageAttachment, TeamAttachment.